_posts/2020-07-31-status-update.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "31th July 2020: IAC Team Status Update"
+author: david22swan
+categories:
+  - team
+  - status
+tags:
+  - releases
+  - pdk
+  - shoutouts
+  - security
+  - rspec-puppet
+  - java
+  - inifile
+  - docker
+---
+
+It's Friday - time for another status update from the IAC Team!
+Here's an overview of some of the work we and the community have been up to recently:
+
+## Module Releases from IAC Team
+We've had a few modules released this week:
+- [puppetlabs-firewall][firewall-forge][`v2.5.0`][firewall-2-5-0]
+- [puppetlabs-websphere_application_server][websphere_application_server-forge][`v2.1.0`][websphere_application_server-2-1-0]
+
+## Community Contributions
+We've had plenty of great community contributions this week:
+- [kenyon][kenyon] [spotted a mistake in the supported list][docker-641] for the [Docker module][docker].
+- [kajinamit][kajinamit] has given us two prs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second set's the [python3-mod_wsgi to be used in place of mod_wsgi on CentOS 8][apache-2052] as the older name no longer works.
```

Comment:
```suggestion
- [kajinamit][kajinamit] has given us two PRs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second sets the [python3-mod_wsgi MOD to be used in place of mod_wsgi on CentOS 8][apache-2052] as the older name no longer works.
```

Reply:
Updated to:
- [kajinamit][kajinamit] has given us two PRs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second sets [python3-mod_wsgi to be used in place of mod_wsgi on CentOS 8][apache-2052] as the older name no longer works.